environment-pipeline.yml

```diff
@@ -0,0 +1,22 @@
+name: longplex-nf
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - bioconda::fastqc=0.12.1
+  - bioconda::lima=2.7.1
+  - bioconda::multiqc=1.21
+  - bioconda::nextflow=23.10.1
```

Comment:
```suggestion
```

Nextflow should be installed _**outside**_ the runtime environment.

Do we need this here?

Local install of Nextflow is still a project pre-req:

https://github.com/seqwell/LongPlex?tab=readme-ov-file#dependencies